packages/subgraph/tests/admin/admin.test.ts

```diff
@@ -105,28 +106,31 @@ test('Run Admin plugin (handleProposalExecuted) mappings with mock event', () =>
   adminPlugin.pluginAddress = Bytes.fromHexString(CONTRACT_ADDRESS);
   adminPlugin.save();
 
-  let entityID =
-    Address.fromString(CONTRACT_ADDRESS).toHexString() +
-    '_' +
-    BigInt.fromString(PROPOSAL_ID).toHexString();
+  let entityID = getProposalId(
+    Address.fromString(CONTRACT_ADDRESS),
+    BigInt.fromString(PROPOSAL_ID)
+  );
 
-  let adminstratorAddress = Address.fromString(ADDRESS_ONE);
+  let administratorAddress = Address.fromString(ADDRESS_ONE);
 
   let adminProposal = new AdminProposal(entityID);
   adminProposal.dao = DAO_ADDRESS;
   adminProposal.plugin = pluginId;
   adminProposal.proposalId = BigInt.fromString(PROPOSAL_ID);
-  adminProposal.creator = adminstratorAddress;
+  adminProposal.creator = administratorAddress;
   adminProposal.metadata = STRING_DATA;
   adminProposal.executed = false;
   adminProposal.createdAt = BigInt.fromString(START_DATE);
   adminProposal.startDate = BigInt.fromString(START_DATE);
   adminProposal.endDate = BigInt.fromString(START_DATE);
   adminProposal.allowFailureMap = BigInt.fromString(ALLOW_FAILURE_MAP);
-  adminProposal.administrator = adminstratorAddress.toHexString();
+  adminProposal.administrator = administratorAddress.toHexString();
   adminProposal.save();
 
-  const actionId = CONTRACT_ADDRESS + '_' + PROPOSAL_ID + '_' + PROPOSAL_ID;
+  const actionId = CONTRACT_ADDRESS.concat('_')
+    .concat(PROPOSAL_ID)
+    .concat('_')
+    .concat(PROPOSAL_ID);
```

Comment:
shouldn't this be the index of the action in the actions array?

Reply:
It seems to me that because of these, tests are unstable. It needs to be written better or completely removed. Dont see any other point.

Reply:
yes, amended